src/aioswitcher/api/packets.py

```diff
@@ -25,13 +25,49 @@
 
 REQUEST_FORMAT_BREEZE = "{}000001000000000000000000{}00000000000000000000f0fe"
 PAD_72_ZEROS = "0" * 72
+P_SESSION = "00000000"
+PHONE_ID = "0000"
+DEVICE_PASS = "00000000"
+
 # format value just timestamp (initial session id is "00000000")
 LOGIN_PACKET_TYPE1 = (
-    "fef052000232a10000000000" + REQUEST_FORMAT_TYPE1[2:] + "{}" + PAD_72_ZEROS + "00"
+    "fef052000232a100"
```

Comment:
Can't figure out better names.. I will try to add some code docs to the file.

Reply:
Done. You can resolve it.

Reply:
Let's add documentation here. The `format values are ...` comment for each packet was OK for one or two types of login packets. Now that we have four, none of them has a meaningful name; let's add some documentation regarding what each login packet is meant for and when to use it.

Reply:
I did that.. it was on my repo for some reason.. please check again

Reply:
Do you mean these?

![image](https://github.com/TomerFi/aioswitcher/assets/28388442/c69690a0-9bef-4b3a-b5b2-16eacc5bdcfc)

I'm sorry, but I'm failing to see what value these add to our already "not-so-clear" naming conventions. I suggest adding a couple of lines explaining what devices qualify as type 1 and type 2 and which use tokens. This is exactly where to add this; it will make life much easier for all contributors.

@thecode If you disagree and ok with this, I can back down on this and mark it as resolved.

Reply:
@YogevBokobza, First, I'm missing the README where this is specified. Please share a link. Second, there's a difference between code documentation (developer-faced) and project documentation (user-faced).